hypershift-operator/controllers/nodepool/nodepool_controller.go

```diff
@@ -350,10 +347,12 @@ func generateScalableResources(client ctrlclient.Client, ctx context.Context,
 	annotations := map[string]string{
 		"machine.cluster.x-k8s.io/exclude-node-draining": "true",
 	}
-	if nodePool.Spec.AutoScaling.Max != nil &&
-		*nodePool.Spec.AutoScaling.Max > 0 {
-		annotations[autoscalerMinAnnotation] = strconv.Itoa(*nodePool.Spec.AutoScaling.Min)
-		annotations[autoscalerMaxAnnotation] = strconv.Itoa(*nodePool.Spec.AutoScaling.Max)
+	if isAutoscalingEnabled(nodePool) {
```

Comment:
use the variable `isAutoscalingEnabled ` created above here?

Reply:
nvm ignore me

Reply:
Not sure I follow, that variable is in another function scope (`reconcile()`)